### DISCUSSION TITLE
Update existing status when running action again

### DISCUSSION THREAD

**Discussion by alexlafroscia** [community/community#25523] [Actions]
<p>We have a GitHub Action set up to run when a PR is either update or edited, which does some linting against the description of the PR (to ensure it references a ticket in our ticketing system).</p>
<p>If a PR is opened without the reference, the GItHub Action reports a failure (which is expected)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/284250a474a5a22dd54f39affc1362428621e7b5.png" data-download-href="/uploads/short-url/5K9b2v8YwxvtUZB144VOOHJ9vU1.png?dl=1" title="Screen Shot 2019-12-13 at 12.35.16 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1645881/181091718-945710f9-8fee-49de-85d1-a7fedb190c28.png" alt="Screen Shot 2019-12-13 at 12.35.16 PM.png" data-base62-sha1="5K9b2v8YwxvtUZB144VOOHJ9vU1" width="690" height="54" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/284250a474a5a22dd54f39affc1362428621e7b5_2_690x54.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/284250a474a5a22dd54f39affc1362428621e7b5_2_1035x81.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/284250a474a5a22dd54f39affc1362428621e7b5_2_1380x108.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/284250a474a5a22dd54f39affc1362428621e7b5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-12-13 at 12.35.16 PM.png</span><span class="informations">1394×110 13 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If the PR description is edited and made to include the ticket reference, it passes the check but adds an <em>additional</em> status about it.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/af22fce2480592e546342b939d6f7303dbbbde11.png" data-download-href="/uploads/short-url/oZkw8K9YFJHwWUosy7N7dO2YbFT.png?dl=1" title="Screen Shot 2019-12-13 at 12.35.21 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1645881/181091719-5c1f727c-40df-48c1-90e1-5ba837f0f25f.png" alt="Screen Shot 2019-12-13 at 12.35.21 PM.png" data-base62-sha1="oZkw8K9YFJHwWUosy7N7dO2YbFT" width="690" height="99" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/af22fce2480592e546342b939d6f7303dbbbde11_2_690x99.png, https://aws1.discourse-cdn.com/github/optimized/2X/a/af22fce2480592e546342b939d6f7303dbbbde11_2_1035x148.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/a/af22fce2480592e546342b939d6f7303dbbbde11_2_1380x198.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/af22fce2480592e546342b939d6f7303dbbbde11_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-12-13 at 12.35.21 PM.png</span><span class="informations">1392×200 23.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Ideally, the previous failure would be cleaned up, as it is no longer relevant.</p>
<p>What’s the best way to make that happen? Is there a way to either</p>
<ol>
<li>Make both executions share the same “output” status?</li>
<li>Remove the original failure once it passes?</li>
</ol>

**Comment by weide-zhou** [comment, marked as answer]
<p>The two check runs are triggered by different pull request event types. 1st by ‘opened’ type, the 2nd by ‘edited’ type.</p>
<p>Currently it’s not support to clean up the 1st check run or group the two check runs.</p>
<p>As an alternative, please use <a href="https://developer.github.com/v3/repos/statuses/" rel="nofollow noopener">api</a> to set status for status check, make it always success then it’s available to get a green check.</p>
<p>Please refer to similar <a href="https://github.community/t/duplicate-checks-on-pull-request-event/16394">ticket</a> for more details.</p>
